front/components/data_source/TableUploadOrEditModal.tsx

```diff
@@ -117,7 +117,7 @@ export const TableUploadOrEditModal = ({
             parentId: undefined,
             parents: undefined,
             async: undefined,
-            csv: undefined,
+            fileId: undefined,
```

Comment:
Passing content is unused, and remains unused. I'd like to use that endpoint instead of the crazy shared data_sources/xxx/files endpoint but that's out of scope